src/pages/products/sgid/society/pre-k-through-12.astro

```diff
@@ -0,0 +1,149 @@
+      found on the <ExternalLink href="https://schools.utah.gov/">Utah State Board of Education website</ExternalLink>.
+    </p>
+    <p>
+      This dataset will be updated annually in conjunction with USBE's <ExternalLink
+        href="https://schools.utah.gov/data/reports">annual report</ExternalLink
+      > cycle.
+    </p>
+    <h3>Methodology</h3>
+    <h4>Tabular Data</h4>
+    <p>
+      The tabular portion of the data was downloaded from USBE's data page. The list of fields and the data in this
+      dataset were generated by joining the following USBE reports:
+    </p>
+    <BulletedList>
+      <li>
+        <ExternalLink href="https://schools.utah.gov/data/reports?mid=1424&tid=6">2019-2020 School List</ExternalLink>
+      </li>
+      <li>
+        <ExternalLink href="https://schools.utah.gov/data/reports?mid=1424&tid=4"
+          >Fall 2019-2020 Enrollment by Demographics and Grade Levels</ExternalLink
+        >
+      </li>
+    </BulletedList>
+    <h4>Geographic Data</h4>
+    <p>
+      Point locations (geometry) were generated from USBE's school list using the <ExternalLink
+        href="http://api.mapserv.utah.gov/#geocoding">UGRC Geocoding Web API</ExternalLink
+      > which utilizes address point and street centerline reference data. These were then compared to the previous 2016-2017
+      school layer's locations with the best position chosen. Unmatched addresses were hand placed with assistance from publicly
+      available, commercial mapping and search web apps.
+    </p>
+    <h3>Fields Worth Noting</h3>
+    <pre>SchoolType</pre>
+    <BulletedList>
+      <li>Residential Treatment</li>
+      <li>Adult High</li>
+      <li>Vocational</li>
+      <li>Special Education</li>
+      <li>Regular Education</li>
+      <li>YIC - <ExternalLink href="https://www.schools.utah.gov/yic">Youth-In-Custody</ExternalLink></li>
+      <li>Alternative</li>
+    </BulletedList>
+
+    <pre>SchoolLevel</pre>
+    <BulletedList>
+      <li>MID - Junior / Middle School</li>
+      <li>HIGH - High School</li>
+      <li>K12 - Kindergarten through 12th Grade</li>
+      <li>PREK - Pre Kindergarten</li>
+      <li>ELEM - Elementary School</li>
+    </BulletedList>
+
+    <h3>Acronyms and Abbreviations in the Data</h3>
+    <p>LEA: Local Educational Agency (i.e., school district)</p>
+    <BulletedList>
+      <li>LEAName</li>
+      <li>LEANumber</li>
+      <li>LEAID</li>
+    </BulletedList>
+
+    <p>NESS: Necessarily Existent Small Schools funding</p>
+
+    <Contacts contactKey="ugrc" />
+  </Fragment>
+
+  <UpdateHistory slot="update-history" updates={page.updateHistory} style="secondary" />
+
+  <Section title="More resources" slot="more-resources" titlePosition="center">
+    <GridForMoreResources>
+      <CardWithPopularLink title="USBE" href="https://schools.utah.gov"
+        >Utah State Board of Education</CardWithPopularLink
+      >
```

Comment:
please keep these as even as possible using sentence case for the title and slot and punctuation in the slot